scripts/ansible/oe-windows-acc-setup.yml

```diff
@@ -8,9 +8,9 @@
   tasks:
     - name: OE setup | Set installer URLs from the OE storage account
       set_fact:
-        intel_psw_url:       "https://oejenkins.blob.core.windows.net/oejenkins/Intel%20SGX%20PSW%20for%20Windows%20v2.5.101.3.exe"
+        intel_psw_url:       "https://oejenkins.blob.core.windows.net/oejenkins/Intel SGX PSW for Windows v2.6.100.2.exe"
```

Comment:
Consider keeping the `%20` encoded spaces as it's generally more robust as the string is getting passed around scripts that may or may not handle spaces correctly.

Reply:
Interesting, I will keep that in mind. Altered and pushed.